packages/oscal-react-library/src/components/OSCALCatalogGroups.test.tsx

```diff
@@ -30,14 +31,30 @@ const testGroups = [
         class: "family",
         title: "Sibling Title",
         controls: [{ id: "control2-id", title: "Audit Events" }],
+        parts: [
+          {
+            name: "Parent part prose",
+            prose: "Prose",
+          },
+        ],
+      },
+    ],
+  },
+  {
+    title: "Test Group C",
+    parts: [
+      {
+        id: "test-c-overview-1",
+        name: "overview",
+        prose: "This group has prose",
       },
     ],
   },
 ];
 
-function getTextByChildren(text) {
-  function result(_content, node) {
-    const hasText = (checkNode) => checkNode.textContent === text;
+function getTextByChildren(text: string) {
+  function result(_content: any, node: any) {
+    const hasText = (checkNode: any) => checkNode.textContent === text;
```

Comment:
I *think* node is a bit of a misnomer here. I think this function is honestly a hacky way to test things because of Catalogs being a bit weird. The `node.children` part is really important and `ReactNode` does not have that attribute. 

I don't love it either, but I also think we may want to clean up catalog groups in the future and make this function unneeded.